### PR TITLE
Mark all tasks as skipped when failing a dag_run manually including t…

### PR DESCRIPTION
**Pull request by utkarsharma2**
(cherry picked from commit eda6a8fcf009eb224ec556f7117a97965dbd4dd5)

Backport of - https://github.com/apache/airflow/pull/43482

### CODE REVIEW

**Comment by Lee-W** [comment]
looks like some test are failing 🤔

File: tests/www/views/test_views_dagrun.py [hunk 1]
```diff
@@ -290,3 +290,79 @@ def test_dag_runs_queue_new_tasks_action(session, admin_client, completed_dag_ru
...
+@pytest.fixture
+def dag_run_with_all_done_task(session):
+    """Creates a DAG run for example_bash_decorator with tasks in various states and an ALL_DONE task not yet run."""
+    dag = DagBag().get_dag("example_bash_decorator")
+
+    # Re-sync the DAG to the DB
+    dag.sync_to_db()
+
+    execution_date = timezone.datetime(2016, 1, 9)
+    triggered_by_kwargs = {"triggered_by": DagRunTriggeredByType.TEST} if AIRFLOW_V_3_0_PLUS else {}
```
Comment (Lee-W) [inline tests/www/views/test_views_dagrun.py]: Test are failing because of `AIRFLOW_V_3_0_PLUS` I think

Comment (utkarsharma2) [reply tests/www/views/test_views_dagrun.py]: Yup, `AIRFLOW_V_3_0_PLUS` doesn't exist in this branch.

Comment (utkarsharma2) [reply tests/www/views/test_views_dagrun.py]: I changed the test for only < 3.0 case.

File: airflow/listeners/test.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import pluggy
```
Comment (Lee-W) [inline airflow/listeners/test.py]: What is this file for ?

Comment (utkarsharma2) [reply airflow/listeners/test.py]: Removed it, was not part of this.

**Review by ephraimbuddy — APPROVED** [review]

**Pull request merged**